### PR TITLE
Added `submodule addrefs` command.

### PR DESCRIPTION
**Pull request by bpeabody**
To make refs in submodules that correspond to refs in the meta-repo.

Partially addresses: https://github.com/twosigma/git-meta/issues/258

### CODE REVIEW

**Comment by novalis** [comment]
Git usually spells "make" as "add".

**Comment by bpeabody** [comment]
Done (but the name of my branch is the same)

File: node/lib/util/submodule_util.js [hunk 1]
```diff
@@ -523,3 +524,64 @@ exports.resolvePaths = co.wrap(function *(workdir,
     return result;
 });
 
+/**
+ * Create references having the specified `refs` names in each of the specified
+ * `submodules`, in the specified `repo` with each created reference being
+ * assigned to the commit indicated for that respective submodule by the ref
+ * with that name in the meta-repo.  Do not create a reference in a submodule
+ * when for references indicating commits in which that submodule does not
```
Comment (novalis) [inline node/lib/util/submodule_util.js]: s/when for/for/

File: node/lib/util/submodule_util.js [hunk 2]
```diff
@@ -523,3 +524,64 @@ exports.resolvePaths = co.wrap(function *(workdir,
...
+ * exist.  Not that if a reference is the *current* branch of a sub-repo, it
```
Comment (novalis) [inline node/lib/util/submodule_util.js]: "Note that"

**Pull request merged**